public/src/js/components/userModalComponent.js

```diff
@@ -7,107 +7,222 @@ var UserModel = require('../models/userModel');
 
 module.exports = React.createClass({
   mixins: [Backbone.React.Component.mixin],
-  
-  attributes: [
-    'idn',
-    'username',
-    'first_name',
-    'last_name',
-    'phone',
-    'website',
-    'github',
-    'is_admin',
-    'is_instructor',
-    'is_student'
-  ],
-
-  componentDidMount: function() {
-    _.each(this.attributes, function (attr) {
-      this.refs[attr].value = this.props.model.get(attr);
-    }, this);
+
+  getInitialState: function() {
```

Comment:
Now that I think of it, the library already creates the state based on the model attributes. You can access them at `this.state.model`. The main purpose of the library is to just glue together `this.state.model` and `this.props.model.attributes`. So we may not need to use `getInitialState()`